### PR TITLE
revoke approvals script

### PR DESCRIPTION
**Pull request by jalbrekt85**
handles #1125 
```boo
brownie run issue/1125/revoke_approvals.py main techops.badgerdao.eth
```
```boo
brownie run issue/1125/revoke_approvals.py main dev.badgerdao.eth
```

### CODE REVIEW

File: scripts/issue/1125/revoke_approvals.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+def main(safe):
+    safe = GreatApeSafe(safe)
+
+    # spenders and tokens involved in dev & techops allowances
+    spenders = [
+        r.sushiswap.routerV2,
+        r.uniswap.NonfungiblePositionManager,
+        r.uniswap.routerV2,
+        "0x4459A591c61CABd905EAb8486Bf628432b15C8b1",  # https://etherscan.io/address/0x4459A591c61CABd905EAb8486Bf628432b15C8b1
+        "0x111111125434b319222CdBf8C261674aDB56F3ae",  # https://etherscan.io/address/0x111111125434b319222CdBf8C261674aDB56F3ae
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/issue/1125/revoke_approvals.py:19]: agree on adding into the book, maybe is worthy having this script generalised in `scripts/badger/` directory for other occasions if needed, so we can plug tokens and spenders whenever

interesting was not aware we ever explicitly trade thru 1inch, but looks quite old

[the remark this answers is not in the document]
Comment (jalbrekt85) [reply scripts/issue/1125/revoke_approvals.py:19]: how would we generate the a list of spenders given an address? originally wanted to make it generalized as well but i couldn't find a good way to do this without just hardcoding it

[the remark this answers is not in the document]
Comment (gosuto-inzasheru) [reply scripts/issue/1125/revoke_approvals.py:19]: yes, without dynamic spender list this doesnt work generalised. maybe revoke has an api or something, but that is a bit outside the scope of this ticket imo

[the remark this answers is not in the document]
Comment (jalbrekt85) [reply scripts/issue/1125/revoke_approvals.py]: added to address book 12d5663

**Review by gosuto-inzasheru — APPROVED** [review]

File: scripts/issue/1125/revoke_approvals.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/issue/1125/revoke_approvals.py]: initially though that revoke has an api, but now checking it is firing up lots of rpcs call instead

**Review by petrovska-petro — APPROVED** [review]

File: scripts/issue/1125/revoke_approvals.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jalbrekt85) [reply scripts/issue/1125/revoke_approvals.py]: ya i was looking to see if they had an api too. unfortunately they do not

**Pull request merged**